File: 241022/장학금/scholarship.cpp
#include <iostream>
using namespace std;

int main() {
    int mid, fin;
    cin >> mid >> fin;

    if (mid >= 90) {
        if (fin >= 95) {
            cout << 100000;
        } else if (fin >= 90) {
            cout << 50000;
        } else {
            cout << 0;
        }
    } else {
        cout << 0;
    }
    return 0;
}

File: 241022/장학금/scholarship.js
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim().split(" ");

let mid = Number(input[0]);
let fin = Number(input[1]);

if (mid >= 90 && fin >= 95) {
    console.log(100000);
} else if (mid >= 90 && fin >= 90) {
    console.log(50000);
} else {
    console.log(0);
}